src/models/sighting/useDeleteSighting.js

```diff
@@ -44,5 +59,7 @@ export default function useDeleteSighting() {
     onClearError: () => setError(null),
     success,
     setSuccess,
+    vulnerableIndividual,
+    setVulnerableIndividual,
```

Comment:
What about `clearVulnerableIndividual` instead? It would better match `onClearError` and for now all you are doing is clearing it anyway.

Reply:
Good point!